### PR TITLE
(chore): Solana guide update

### PR DESCRIPTION
**Pull request by aditya520**
This is a part of the pricefeed how-to guide refactor.

In this PR, I just added some visibility to important text based on devs feedback and @guibescos comments.

### CODE REVIEW

File: pages/price-feeds/use-real-time-data/solana.mdx [hunk 1]
```diff
@@ -71,17 +75,25 @@ pub fn sample(ctx: Context<Sample>) -> Result<()> {
...
+  **NOTE**: Users must validate the price update for the appropriate **price
+  feed** and **timestamp**. `PriceUpdateV2` guarantees that the account contains
+  a verified price for _some_ price feed at _some_ point in time. There are
+  various methods on this struct (such as `get_price_no_older_than`) that users
+  can use to implement the necessary checks.
+</Callout>
+
+<Callout type="info" emoji="ℹ️">
+  **Note**: Use an account address check to validate the price feed ID, if a
+  user wants to integrate using the price feed accounts (see below).
```
Comment (guibescos) [inline pages/price-feeds/use-real-time-data/solana.mdx:88]: I like the old text better : 
`if you choose the price feed account integration (see below), you can use an account address check to validate the price feed id.`

File: pages/price-feeds/use-real-time-data/solana.mdx [hunk 2]
```diff
@@ -115,16 +127,16 @@ const solUsdPriceFeedAccount = pythSolanaReceiver
   .toBase58();
 ```
 
-The price feed account integration assumes that an off-chain process is continuously updating each price feed.
-The Pyth Data Association sponsors price updates for a subset of commonly-used price feeds on shard 0.
+**NOTE:** The Price Feed Accounts integration assumes that an off-chain process is continuously updating each price feed.
```
Comment (guibescos) [inline pages/price-feeds/use-real-time-data/solana.mdx:130]: Notes seem inconsistent. 
Sometimes it's written as NOTE sometime as Note.
Sometimes it's inside a Callout and sometimes not.

File: pages/price-feeds/use-real-time-data/solana.mdx [hunk 3]
```diff
@@ -189,7 +203,7 @@ await transactionBuilder.addPriceConsumerInstructions(
   }
 );
 
-// Send the instructions in the builder in 1 or more transactions.
+// Send the instructions to the builder in 1 or more transactions.
```
Comment (guibescos) [inline pages/price-feeds/use-real-time-data/solana.mdx:206]: The old text is more accurate, we have already added to the instruction to the builder at this point, and we just need to send them.

File: pages/price-feeds/use-real-time-data/solana.mdx [hunk 4]
```diff
@@ -115,16 +127,21 @@ const solUsdPriceFeedAccount = pythSolanaReceiver
...
 ```
 
-The price feed account integration assumes that an off-chain process is continuously updating each price feed.
-The Pyth Data Association sponsors price updates for a subset of commonly-used price feeds on shard 0.
-Please see [Sponsored Feeds](/price-feeds/sponsored-feeds) for a list of sponsored feeds and their account addresses.
-However, updating a price feed is a permissionless operation, and anyone can run this process.
-Please see [Using Scheduler](/price-feeds/schedule-price-updates/using-scheduler) for more information.
-Running the scheduler can help with reliability and to update feed/shard pairs that are not part of the default schedule.
+<Callout type="info" emoji="ℹ️">
+  **NOTE:** The Price Feed Accounts integration assumes that an off-chain
```
Comment (jayantk) [inline pages/price-feeds/use-real-time-data/solana.mdx:131]: Whiy is Price Feed Account capitalized here. It's not capitalized above. I suggest not capitalizing.

File: pages/price-feeds/use-real-time-data/solana.mdx [hunk 5]
```diff
@@ -115,16 +127,21 @@ const solUsdPriceFeedAccount = pythSolanaReceiver
...
+  process is continuously updating each price feed. The Pyth Data Association
+  sponsors price updates for a subset of commonly used price feeds on shard 0.
+  Please see [Sponsored Feeds](/price-feeds/sponsored-feeds) for a list of
+  sponsored feeds and their account addresses. However, updating a price feed is
```
Comment (jayantk) [inline pages/price-feeds/use-real-time-data/solana.mdx:135]: Suggest a paragraph break before "However", and maybe changing it to say "Additionally, updating..."

**Review by jayantk — APPROVED** [review]
Nice. Left a couple minor comments.

File: pages/price-feeds/use-real-time-data/solana.mdx [hunk 6]
```diff
@@ -71,17 +75,25 @@ pub fn sample(ctx: Context<Sample>) -> Result<()> {
...
+  **NOTE**: Users must validate the price update for the appropriate **price
+  feed** and **timestamp**. `PriceUpdateV2` guarantees that the account contains
+  a verified price for _some_ price feed at _some_ point in time. There are
+  various methods on this struct (such as `get_price_no_older_than`) that users
+  can use to implement the necessary checks.
+</Callout>
+
+<Callout type="info" emoji="ℹ️">
+  **NOTE**: If you choose the price feed account integration (see below), you
+  can use an account address check to validate the price feed id.
```
[the remark this answers is not in the document]
Comment (aditya520) [reply pages/price-feeds/use-real-time-data/solana.mdx:88]: I believe it's important, so 2 separate callouts grabs more attention.

[the remark this answers is not in the document]
Comment (aditya520) [reply pages/price-feeds/use-real-time-data/solana.mdx:88]: But yeah, it looks weird.

**Pull request merged**